playbooks/ran/configure-lvm-storage.yml

```diff
@@ -0,0 +1,135 @@
+---
+# This playbook configures LVM storage on OpenShift master nodes and creates
+# LocalVolume resources for local storage operator.
+#
+# Prerequisites:
+#   - SSH access to master nodes
+#   - Valid kubeconfig for cluster authentication
+#   - Local Storage Operator must be installed
+#
+# Usage:
+#   ansible-playbook -i inventories/ocp-deployment/build-inventory.py \
+#     playbooks/configure-lvm-storage.yml \
+#     --extra-vars "kubeconfig=/path/to/kubeconfig"
+#
+# What it does:
+#   1. Creates LVM volume group and logical volumes on master nodes
+#   2. Creates OpenShift LocalVolume CRs to expose the volumes to the cluster
+#   3. Sets local-sc as the default storage class
+
+- name: Configure LVM storage on master nodes
+  hosts: masters
+  become: true
+  remote_user: core
+  gather_facts: false
+
+  tasks:
+    - name: Create volume group new_vol_group on /dev/vda
+      community.general.lvg:
+        vg: new_vol_group
+        pvs: /dev/vda
+        state: present
+
+    - name: Create logical volume vol1 (20GB)
+      community.general.lvol:
+        vg: new_vol_group
+        lv: vol1
+        size: 20G
+        state: present
+
+    - name: Create logical volume vol2 (20GB)
+      community.general.lvol:
+        vg: new_vol_group
+        lv: vol2
+        size: 20G
+        state: present
+
+- name: Create LocalVolume resources on bastion
+  hosts: bastion
+  gather_facts: false
+  environment:
+    K8S_AUTH_KUBECONFIG: "{{ kubeconfig }}"
+
+  tasks:
+    - name: Validate kubeconfig variable is defined
+      ansible.builtin.assert:
+        that:
+          - kubeconfig is defined
+          - kubeconfig | length > 0
+        fail_msg: |
+          Variable 'kubeconfig' must be defined.
+          Pass it with --extra-vars 'kubeconfig=/path/to/kubeconfig'
```

Comment:
we should start with those checks before

Reply:
I know but, for best practice you should first validated the pre-requirement 

also why we should run stuff if it will fail because of a missing var - it's not efficient workflow